.editorconfig

```diff
@@ -0,0 +1,11 @@
+root = true
+
+[*]
+charset = utf-8
+insert_final_newline = true
+indent_style = space
+indent_size = 2
+trim_trailing_whitespace = true
+
+[*.md]
+trim_trailing_whitespace = false
```

Comment:
Yeah, that's why I usually add it to PRs when making them. IDEs really enjoy strpping that whitespace...